duckduckgo/Android
# Tabs

Reviewer: @subsymbolic 
Asana:https://app.asana.com/0/0/350058423386223

**Description**: Add tabs to the project


**Steps to test this PR**:
1. Use the tab switcher to delete, select or create new tabs.
2. Ensure that each tab keep its state when switching, and is removed when deleted.
3. Check that the app keep its state on configuration change and app recreation after the process is killed, but is rebooted when swiped from the recent app.


###### Reviewer Checklist:


###### PR DRI Checklist:
